framework/src/application/node/transport/broadcaster.js

```diff
@@ -17,28 +17,23 @@
 const ENDPOINT_BROADCAST_TRANSACTIONS = 'postTransactionsAnnouncement';
 
 class Broadcaster {
-	constructor({ broadcasts, transactionPool, logger, channel }) {
+	constructor({ transactionPool, logger, channel, releaseLimit, interval }) {
+		this.channel = channel;
 		this.logger = logger;
 		this.transactionPool = transactionPool;
-		this.config = broadcasts;
-		this.channel = channel;
+		this.releaseLimit = releaseLimit;
+		this.interval = interval;
 
 		this.queue = [];
 		this.transactionIdQueue = [];
 
-		if (this.config.active) {
```

Comment:
Does the broadcasting will always be active?

Reply:
yes, it will always active, and broadcasting disable was only for testing purpose.
Also, for rebuilding, the braodcast event is not registered, so it will not broadcast